static/main.ts

```diff
@@ -22,42 +22,52 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-'use strict';
```

Comment:
Looks like there were no type errors in the js version judging by the diff, which is pretty unexpected seeing how the rest of the TS conversions have gone down, so I'm pleasantly supprised :)